### PR TITLE
add instanceevent to machineimage relationship

### PR DESCRIPTION
**Pull request by infinitewarp**
Related to https://github.com/cloudigrade/cloudigrade/issues/355, when building the queries to gather the necessary data, I realized that we really should have established a relationship between from the `InstanceEvent` to the `MachineImage` to indicate what image was seen at the time of the event. This PR does exactly that, and it migrates the old (and now redundant) `AwsInstanceEvent.ec2_ami_id` in the process.

### CODE REVIEW

**Comment by infinitewarp** [comment]
@cloudigrade/quality-engineering This change removes `ec2_ami_id` from the API that lists the instance events, and it replaces with a link to the image. If you guys have any tests that assert on that field specifically, we will need to update those...

File: cloudigrade/account/util.py [hunk 1]
```diff
@@ -71,31 +74,48 @@ def save_instance_events(account, instance_data, region, events=None):
...
                 occurred_at=event['occurred_at'],
                 subnet=event['subnet'],
-                ec2_ami_id=event['ec2_ami_id'],
                 instance_type=event['instance_type'],
             )
             event.save()
     return instance
 
 
+# def get_or_create_awsmachineimage(account, ec2_ami_id):
```
Comment (abaiken) [inline cloudigrade/account/util.py]: why is this commented out?

Comment (infinitewarp) [reply cloudigrade/account/util.py]: Good catch. This is leftover from a previous experiment. Will delete in the next commit!

[the remark this answers is not in the document]
Comment (infinitewarp) [reply cloudigrade/account/util.py]: Good catch. This is leftover from a previous experiment. Will delete in the next commit!

**Pull request merged**